jbpm-installer/src/test/java/org/jbpm/persistence/scripts/util/TestsUtil.java

```diff
@@ -62,6 +62,11 @@ public int compare(File o1, File o2) {
                         return o1.getName().compareTo(o2.getName());
                     }
                 });
+                
+                if (databaseType.equals(DatabaseType.POSTGRESQL)) {
+                    //Returns first schema sql
+                    Arrays.sort(foundFiles, Comparator.<File, Boolean>comparing(s -> s.getName().contains("schema")).reversed());
+                }
```

Comment:
Is there any need for this sorting? What happens if we run `lo-trigger-clob` and `quartz` scripts before the schema script? Right now the order should be:
`postgresql-jbpm-lo-trigger-clob.sql`
`postgresql-jbpm-schema.sql`
`quartz_tables_postgres.sql`

and everything works fine.

Reply:
With that order, following exception is thrown, because that table is created in the "postgresql-jbpm-schema.sql":
`ERROR: relation "booleanexpression" does not exist`

Reply:
Ah, OK, makes sense.